Assignment2/hunter097-1.b.ttl

```diff
@@ -0,0 +1,18 @@
+@base < www.oeg upm.net / person >
+@prefix : < www.oeg upm.net / person #>
+@prefix person : < www.oeg upm.net / person #>
+
+:Class a rdfs:Class.
+	
+:Sensor029 a rdfs:Class;
+	rdfs:subClassOf:Class.
+
+:Computer101 a rdfs:Class;
+	rdfs:subClassOf:Class.

```

Comment:
Wrong syntax and bad use of prefix rdfs. Same below.

Reply:
I think undertood